lib/Dialect/TritonGPU/IR/LinearLayoutConversions.cpp

```diff
@@ -398,6 +398,44 @@ LinearLayout hopperMmaToLinearLayout(ArrayRef<int64_t> shape,
   return combineCtaCgaWithShape(ctaLayout, mma.getCTALayout(), shape);
 }
 
+std::optional<LinearLayout> mfmaToLinearLayout(ArrayRef<int64_t> shape,
```

Comment:
I know this is still a WIP, so feel free to ignore this.  But what does it mean to return nullopt here?  Does it mean the layout is illegal?  In that case we should assert and ideally add an IR verifier.  Or does it mean the transformation to linear layout is not yet supported?

Reply:
At this moment I consider this layout illegal.
We can define smaller MFMA layout, we just did not need it before.

I'll see how to verify this property.

Reply:
https://github.com/triton-lang/triton/pull/4117 to address this.